README.md

```diff
@@ -1,2 +1,140 @@
 # postgres-queries
-Query builder for PostgreSQL in Node.js, built for async
+
+Utility to generate and execute postgresql queries with ease.
+
+## Install
+
+`npm install --save postgres-queries`
+
+## Introduction
+Creating query and executing them are two separate concerns. And thus postgres-queries is divided in two parts:
+- The pool, that allows to connect to the postgres database and execute query.
+- The query builders (insertOne, selectOne, etc..) that allows to generate sql, and the corresponding parameters.
+
+## Documentation
+go to [https://marmelab.com/postgres-queries/](https://marmelab.com/postgres-queries/)
+
+## Pool
+Extend [node-pg-pool](https://github.com/brianc/node-pg-pool)
+Allow to connect to postgresql and execute query
+It adds:
+- namedQuery
+- a link helper to link a query builder to the client.
+
+## Query Builder
+The main idea behind the query builder is to be able to just give a configuration object:
+- the name of a table
+- its primary key(s)
+- the columns we want to read
+- the columns we want to write
+
+And get a function asking for the query parameter:
```

Comment:
It is the result of the query builder?
If yes, I suggest `The builder returns a function which...`

If no, I didn't understand this part.